### ISSUE TITLE
DateTimePicker on Raspberry PI 3

### ISSUE THREAD

**Issue by Killerloop2** [mono/mono#18570]
## Steps to Reproduce

1. Add a DateTimePicker to your Windows Form project 
2. Change the Thread Culture to anything other than "en-US", I used "it-IT"
3. Open the Calendar at runtime

### Current Behavior

The calendar popup will change language but will show "Today:" in the lower section regardless of the chosen culture and language. 

### Expected Behavior

The calendar popup should display "Today:" in the chosen language, for example "Oggi:" in Italian

## On which platforms did you notice this

[ ] Raspberry PI 3

**Version Used**:

6.8.0.96 stable

**Comment by abrevet-dev** [comment]
https://github.com/mono/mono/blob/master/mcs/class/System.Windows.Forms/System.Windows.Forms/ThemeWin32Classic.cs#L3660

**Comment by Killerloop2** [comment]
Yes exactly, that should be replaced with "Today" translated in the current culture

**Comment by abrevet-dev** [comment]
Looks like DateTimePicker will always use the OS culture, not the one of your app.

**Comment by Killerloop2** [comment]
I think that Mono has the better approach

**Comment by Killerloop2** [comment]
Is it some sort of fix expected?
